src/test/java/org/folio/rest/impl/TemplateDeletionTest.java

```diff
@@ -71,8 +71,11 @@ static void beforeAll(Vertx vertx, VertxTestContext context) throws IOException
     PostgresClient.getInstance(vertx).startEmbeddedPostgres();
```

Comment:
All code that configures and starts and stops the database for testing should be removed. PostgresClient was updated to do this automatically: https://github.com/folio-org/raml-module-builder#postgresql-integration